src/sentry/issues/query.py

```diff
@@ -0,0 +1,3 @@
+def apply_performance_conditions(conditions, group):
```

Comment:
I want to figure out how to centralize this logic better. We're doing this kind of thing in a lot of places, but in slightly different ways.